.github/workflows/ruby-core.yml

```
@@ -50,8 +50,13 @@ jobs:
         with:
           path: ruby/rdoc
       - name: Build RDoc locally
+        # The `build` task provided by `require 'bundler/gem_tasks'` seems to have a bug
+        # as described in https://github.com/rubygems/rubygems/issues/8477
+        # The manual `gem build` and `gem install` are used to work around this issue.
         run: |
           bundle install
+          gem build rdoc.gemspec
+          gem install rdoc-*.gem
```

How about adding a note that the installed gem isn't used for generating Ruby documentation? (Should we see the linked issue?)

```suggestion
          # This gem isn't used for generating Ruby documentation.
          # This is just for fixing `pkg/rdoc-X.Y.Z.gem` path.
          gem install rdoc-*.gem
```